test-tool/test_write10_residuals.c

```diff
@@ -25,16 +25,34 @@
 #include "iscsi-private.h"
 #include "scsi-lowlevel.h"
 #include "iscsi-test-cu.h"
-
+#include "test_write_residuals.h"
 
 void
 test_write10_residuals(void)
 {
-        struct scsi_task *task_ret;
-        unsigned char buf[10000];
-        struct iscsi_data data;
-        int ok;
-        unsigned int i;
+        /* testing scenarios */
+        const struct residuals_test_data write10_residuals[] = {
```

Comment:
Please change 'const' into 'static const'.

Reply:
I am using `block_size` variable to initialize this structure. Therefore I can not make it static.